packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -24,6 +24,7 @@ export const messages = {
   source_missing_text: '%s points to a missing folder',
   java_runtime_missing_text:
     'Java runtime could not be located. Set one using the salesforcedx-vscode-apex.java.home VS Code setting.',
+  force_apex_test_run_codeAction_description_text: 'Run Apex test(s)',
```

Comment:
@ruthemmanuelle - Text review?

Reply:
<img width="485" alt="screen shot 2018-08-02 at 12 57 11 pm" src="https://user-images.githubusercontent.com/39964061/43604759-d1933302-9653-11e8-844b-dff40aaf84d9.png">
This is shown as a loading bar when the process is run. I can definitely change it to not have the (s), but I did grab this from other run test commands in the extension (just a heads up).

Reply:
Got it. Yeah, I think we talked about this decided that following the style guideline for the other run test commands would cause confusion. I think this is OK how it is. Thanks for the screenshot.